### DISCUSSION TITLE
Colored Inputs for Skinning

### DISCUSSION THREAD

**Discussion by CuredDDN** [ppy/osu#29462] [Ideas]
Saw this youtube comment
![Screenshot_20240816_154108_YouTube](https://github.com/user-attachments/assets/de59b465-5f31-4b47-abe5-986108b913be)
And thought it was a nice idea to be shared

I do think that the last taken input should be shown incase two inputs are pressed

**Comment by PercyDan54** [comment]
It would require some amount of work since nothing about the input device is store in replay currently
